pom.xml

```diff
@@ -207,6 +207,11 @@
     <commons-cli.version>1.5.0</commons-cli.version>
     <bouncycastle.version>1.60</bouncycastle.version>
     <tink.version>1.6.1</tink.version>
+    <!-- When upgrading `netty.version`, need to check whether
+         the version of `netty-tcnative-classes.version` also needs to be upgraded
```

Comment:
Better use `netty-bom` to manage this for you - https://search.maven.org/artifact/io.netty/netty-bom/4.1.76.Final/pom

Reply:
`netty-bom` with `exclusions`?

Reply:
No. `-bom`s are included in `<dependencyManagement>` section and they just define defaults.
See https://github.com/apache/avro/commit/387f497285d49ffc577263d154bca968e2b5ea65 for example with Jackson.
The idea is that you **import** the BOM in the parent pom.xml and then in the children Maven modules you declare the dependencies without specifying **version**, it is managed by the BOM.

Reply:
Thanks ~ let me try it, should make the configuration look simpler

Reply:
Hmm... I remember that I use `netty-all` here to override the dependency of `hadoop-hdfs-2.7.4` on `netty-all-4.0.23.Final`. 

If `netty-bom` is used, maybe also need to configure 'hadoop-hdfs' separately in `hadoop-2` profile and exclude `netty-all`.